neuraxle/checkpoints.py

```diff
@@ -363,6 +432,80 @@ def checkpoint_exists(self, checkpoint_path, current_id) -> bool:
         return True
 
 
+class TextSummaryCheckpointer(BaseSummaryCheckpointer):
+    """
+    Summary Checkpointer that uses a txt file to create a summary file that contains the list of all of the checkpoint current ids.
```

Comment:
@alexbrillant I want to understand why this class needs to save all the current ids rather than appending new summary ids to an existing list of summary ids.

Reply:
@guillaume-chevalier the summary file needs to contain all of the current ids because we need to know which current ids were saved during the checkpoint. when resuming, we don't know what the current ids were. we need to load each current ids inside the summary file when resuming. not sure what you had in mind. I thought we understood each other here. your question worries me.